python/tests/integration/connectors/memory/test_azure_cosmosdb_memory_store.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from datetime import datetime
+
+import numpy as np
+import pytest
+
+from semantic_kernel.connectors.memory.azure_cosmosdb.azure_cosmos_db_memory_store import (
+    AzureCosmosDBMemoryStore,
+)
+from semantic_kernel.memory.memory_record import MemoryRecord
+from semantic_kernel.memory.memory_store_base import MemoryStoreBase
+
+index_name = "sk_test_vector_search_index"
+vector_dimensions = 1536
+num_lists = 1
+similarity = "COS"
+collection_name = "sk_test_collection"
+database_name = "sk_test_database"
+
+
+def create_embedding(non_zero_pos: int) -> np.ndarray:
+    # Create a NumPy array with a single non-zero value of dimension 1546
+    embedding = np.zeros(vector_dimensions)
+    embedding[non_zero_pos - 1] = 1.0
+    return embedding
+
+
+@pytest.fixture
+def memory_record1():
+    return MemoryRecord(
+        id="test_id1",
+        text="sample text1",
+        is_reference=False,
+        embedding=create_embedding(non_zero_pos=1),
+        description="description",
+        additional_metadata="additional metadata",
+        external_source_name="external source",
+        timestamp=datetime.now(),
+    )
+
+
+@pytest.fixture
+def memory_record2():
+    return MemoryRecord(
+        id="test_id2",
+        text="sample text2",
+        is_reference=False,
+        embedding=create_embedding(non_zero_pos=2),
+        description="description",
+        additional_metadata="additional metadata",
+        external_source_name="external source",
+        timestamp=datetime.now(),
+    )
+
+
+@pytest.fixture
+def memory_record3():
+    return MemoryRecord(
+        id="test_id3",
+        text="sample text3",
+        is_reference=False,
+        embedding=create_embedding(non_zero_pos=3),
+        description="description",
+        additional_metadata="additional metadata",
+        external_source_name="external source",
+        timestamp=datetime.now(),
+    )
+
+
+async def azurecosmosdb_memorystore() -> MemoryStoreBase:
+    store = await AzureCosmosDBMemoryStore.create(
```

Comment:
will this run in CI? I don't see updates to the integration test config

Reply:
No, we want this to be ignored. Let me look into this.

Reply:
Added a conditional to disable the test from running